paper-input-behavior.html

```diff
@@ -180,6 +180,13 @@
       return this.$.input;
     },
 
+    /**
+     * Validates the input element and sets an error style if needed.
+     */
+     validate:function () {
+       return this.querySelector('paper-input-container').validate(this.inputElement, true);
```

Comment:
It seems a little weird to me to have a `validate` function on `<paper-input-container>`, since it doesn't have validatable content but reflects the state of a validatable thing in its light DOM.

What do you think about calling `validate` on the input, which sets `invalid` (the property needs a `notify:true` in `iron-validatable-behavior`), and use binding to set the `invalid` property on the `<paper-input-container>`?